DesktopUI2/DesktopUI2/ViewModels/AccountViewModel.cs

```diff
@@ -113,7 +113,8 @@ public void DownloadImage(string url)
       }
       catch (Exception ex)
       {
-
+        Serilog.Log.ForContext("imageUrl", url)
+          .Debug(ex, "Swallowing exception {exceptionMessage}",ex.Message);
```

Comment:
All the Debug events are retained for a short period of time (3 days) for now.

I'd make all of these "swallows" a warning level for now, so that we can better diagnose stuff.